custom_components/zaptec/zaptec/api.py

```
@@ -96,12 +97,12 @@ def __iter__(self) -> Iterator[str]:
     @property
     def id(self) -> str:
         """Return the id of the object."""
-        return self._attrs["id"]
+        return str(self._attrs["id"])
```

Can `self._attrs["id"]` ever be anything else than `str`? While I understand this helps with the type, it will return the string "None" if the object is `None`. Should it rather raise an exception if the object is non-str? -- I'm not sure what's best here, so I'm open to a discussion.